### ISSUE TITLE
Binary SCAN files

### ISSUE THREAD

**Issue by blairdrummond** [dacostalab/scbursts#25]
Look into making `scan.read` autodetect and read in text or binary files.

**Comment by mdext070** [comment]
After reviewing the scn2txt.py script from DCPROGS, which writes SCAN results to a text file, I think we will abandon this endeavour. The reason is that the scn2txt.py script is a mile long, and I don't have the time to translate all of those functions to R. I think we will just assume that users of SCAN will know how to convert their results to text using DCPROGS' scripts.

**Comment by blairdrummond** [comment]
Fair enough. In lieu of that, can we add a short error message for anyone who tries to read a binary scan file? Like "Please translate this via DCPROGS first"?

**Comment by mdext070** [comment]
Will do.
